manifests/07_deployment.yaml

```diff
@@ -69,6 +70,5 @@ spec:
         effect: "NoSchedule"
       securityContext:
```

Comment:
How about we move the other two (`runAsNonRoot` and `seccompProfile`) into container `securityContext` as well and remove `podSecurityContext` here so it's on one place only?

Unlike `readOnlyRootFilesystem` those two can be set on both places (container taking precedence) and are inherited by the containers from pod context - so it does not make a difference in how it works really. But seems a bit easier to understand as we have only one container now and there is no reason to have it on pod level.